modules/nodes-nodes-audit-log-basic-config.adoc

```diff
@@ -0,0 +1,112 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-audit-log.adoc
+
+[id="nodes-nodes-audit-log-basic-config-{context}"]
+= Configuring the Basic Audit Log
+
+
+To configure the basic audit....
+
+//This is a best guess attempt inspired by https://github.com/openshift/api/pull/279
+
+.Prerequisite
```

Comment:
So, the basic functionality of auditing, requires something (and operator) that is no longer part of the core product?